File: server/src/main/java/com/example/appserver/model/InstrumentCategories.java
package com.example.appserver.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "InstrumentCategories")
public class InstrumentCategories {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private int instrumentCategoryID;

    private String instrumentCategoryName;
    private String pngUrl;
}

File: android/app/src/main/java/com/example/app/model/InstrumentCategories.kt
package com.example.app.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class InstrumentCategories(
    @PrimaryKey(autoGenerate = true) val instrumentCategoryID: Int,
    val instrumentCategoryName: String,
    val pngUrl: String
    ) {}